docs/manuals/source/IntroductionAndTutorial/InstallingBareosWebui.rst

```diff
@@ -62,9 +62,9 @@ System Requirements
 
 -  The |webui| can be installed on any host. It does not have to be installed on the same as the |dir|.
 
--  The default installation uses an Apache webserver with mod-rewrite, mod-php and mod-setenv.
+-  PHP-FPM with Apache HTTP webserver having mod-rewrite enabled.
```

Comment:
```suggestion
-  The default installation uses PHP-FPM with Apache HTTP webserver having mod-rewrite enabled.
```

Reply:
How about upgrading from bareos-21 with mod_php to bareos-22 with php-fpm? 
What needs to be done here? 
Has that been tested?
We should definitely have a documentation what needs to be done AND a breaking changes info in CHANGELOG.md

Reply:
I agree, we should provide a section `Upgrade from 21.x to 22.x` in our documentation in the `Installing Bareos Webui` chapter. Currently testing upgrades and gathering information, will push a commit on top when done.

Reply:
Applied, see https://github.com/bareos/bareos/pull/1287/commits/da48955442b440c0892f8862c65fa795c4a23872 .